<launch>
    <param name="/use_sim_time" value="true" type="boolean"/>
    <arg name="bags_path" doc="path to bag files" default="$(env HOME)/bags" />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find tracking)/rviz/obstacle_estimation_test_sea.rviz" required="true"/>
    <node pkg="rosbag" type="play" name="rosbag" args="--start=40 -r 1 --duration=15 $(arg bags_path)/Na/20180425_Pruebas/2018-04-25-12-57-04.bag" required="true"/>

    <include file="$(find detection)/launch/sea.launch"/>
    
    <node pkg="tracking" type="estimator_node" name="estimator_node" output="screen" required="true">
        <param name="estimator_rate" value="5.0" type="double" />
        <param name="lost_time_th" value="2.5" type="double" />
        <!--<param name="min_update_count" value="3" type="int" /> -->
        <param name="association_th" value="7.0" type="double" /> <!-- Max distance to accept a match between target and candidate -->
        <param name="delay_max" value="2.0" type="double" />
        <param name="use_mahalanobis_distance" value="false" type="bool" />
        <param name="reference_frame"  value="/velodyne" type="str" />
        <remap from="candidates" to="candidates"/>
        <remap from="targets_belief" to="targets_belief"/>
    </node>
</launch>